database/mysql/build-demo.sh

```
@@ -0,0 +1,38 @@
+
+set -euo pipefail
+SCRIPT_DIR="$(cd "$(dirname "$0")" && pwd)"
+IN="${1:?usage: build-demo.sh <input-full-demo.sql> <output-filtered.sql>}"
+OUT="${2:?usage: build-demo.sh <input-full-demo.sql> <output-filtered.sql>}"
+
+# Live tables = the CREATE TABLE names across the baseline schema files (common + both provinces).
+grep -hoE 'CREATE TABLE +`[^`]+`' \
+  "${SCRIPT_DIR}/migration/common/V1__baseline_schema.sql" \
+  "${SCRIPT_DIR}/migration/on/V1.0.1__on_schema.sql" \
+  "${SCRIPT_DIR}/migration/bc/V1.0.1__bc_schema.sql" \
+  | sed -E 's/CREATE TABLE +`([^`]+)`/\1/' | LC_ALL=C sort -u > /tmp/_live_tables.txt
```

Shellcheck (0.11.0)

[info] Expressions don't expand in single quotes, use double quotes for that.

(SC2016)